modules/installation-vsphere-config-yaml.adoc

```diff
@@ -126,28 +157,23 @@ accounts for the dramatically decreased machine performance.
-<5> The number of control plane machines that you add to the cluster. Because
+<5> `replicas`: The number of control plane machines that you add to the cluster. Because
 the cluster uses this values as the number of etcd endpoints in the cluster, the
 value must match the number of control plane machines that you deploy.
-<6> The cluster name that you specified in your DNS records.
-<7> The fully-qualified hostname or IP address of the vCenter server.
-<8> The name of the user for accessing the server. This user must have at least
-the roles and privileges that are required for
-link:https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/vcp-roles.md[static or dynamic persistent volume provisioning]
-in vSphere.
-<9> The password associated with the vSphere user.
-<10> The vSphere datacenter.
-<11> The default vSphere datastore to use.
-<12> Optional: For installer-provisioned infrastructure, the absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID. If you are providing the infrastructure for the cluster, omit this parameter.
-<13> Optional: For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/example_datacenter/host/example_cluster/Resources`.
-<14> The vSphere disk provisioning method.
+<6> `name`: The cluster name that you specified in your DNS records.
+<7> `datacenter`: The vSphere datacenter.
+<8> `resourcePool`: Optional parameter. For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/example_datacenter/host/example_cluster/Resources`.
+<9> `folder`: Optional parameter For installer-provisioned infrastructure, the absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID. If you are providing the infrastructure for the cluster, omit this parameter.
+<10> `vcenters`: The fully-qualified hostname or IP address of the vCenter server.
```

Comment:
`vcenters` is a list, `<10>` should be on `server:`

Reply:
:heavy_check_mark: